ydb/core/fq/libs/compute/ydb/control_plane/compute_database_control_plane_service.cpp

```diff
@@ -374,7 +416,38 @@ class TComputeDatabaseControlPlaneServiceActor : public NActors::TActorBootstrap
         Register(new TCreateDatabaseRequestActor(Clients, SynchronizationServiceActorId, Config, ev));
     }
 
+    void Handle(TEvYdbCompute::TEvCpuLoadRequest::TPtr& ev) {
+        auto actorId = GetMonitoringActorIdByScope(ev.Get()->Get()->Scope);
+        if (actorId != TActorId{}) {
```

Comment:
```
        if (actorId != TActorId{}) {
            Send(ev->Forward(actorId));
            return;
        }
        Send(ev->Sender, new TEvYdbCompute::TEvCpuLoadResponse(NYql::TIssues{NYql::TIssue{TStringBuilder{} << "Cluster load monitoring disabled"}}), 0, ev->Cookie);
```

to get rid of `else`

Reply:
Why it is better?

Reply:
does not matter in this case imho
but yes, no "else" - simplier code